packages/react/src/components/Footer/Footer.js

```diff
@@ -58,7 +58,8 @@ const Footer = ({ type, navigation, langCode, disableLocaleButton }) => {
   useEffect(() => {
     (async () => {
       const response = await LocaleAPI.getLangDisplay(langCode);
-      setDisplayLang(response);
+      const formatedLangDisplay = response.replace('-', '—');
```

Comment:
This should actually be replaced in the LocaleAPI service itself. It is in the services package in the same repo.